main.go

```diff
@@ -18,15 +19,18 @@ import (
 // Version holds the current Gitea version
 var Version = "1.1.0+dev"
 
+// Tags holds the build tags used
+var Tags = ""
+
 func init() {
-	setting.AppVer = Version
+	setting.AppVer = Version + formatBuiltWith(Tags)
```

Comment:
This should be done with a separate variable, IMHO we should not list the build tags on every page in the footer, only within the admin dashboard